app/models/principal.rb

```diff
@@ -178,4 +176,7 @@ def set_default_empty_values
     self.mail ||= ''
     true
   end
+
+  extend Pagination::Model
```

Comment:
This move allows to determine whether the model already defined a `like` scope / method.